### PR TITLE
paths: Display value on card

### PR DESCRIPTION
**Pull request by EDsCODE**
## Changes

*Please describe.*  
- provides the total number of people that made it to a path node on the card directly
- the value will have to be moved on hover in order to provide menu for options
*If this affects the frontend, include screenshots.*  
Screen Shot 2021-09-28 at 7 40 48 PM

## How did you test this code?

*Please describe.*
Manual QA since it's a small frontend change

### CODE REVIEW

**Comment by liyiy** [comment]
Hmm would this make it even more unintuitive for people to know about the more options dropdown? I feel like
Screen Shot 2021-09-28 at 8 01 26 PM
already has a difficult time finding space to include the dropdown and having the people count display and then change into a `...` dropdown on hover seems a little weird?

**Comment by EDsCODE** [comment]
Fair. @clarkus i think we should yeet the step index and put the people number there because im finding the number to be really useful immediately. Maybe add index information in the dropdown somewhere if at all

**Comment by clarkus** [comment]
> Fair. @clarkus i think we should yeet the step index and put the people number there because im finding the number to be really useful immediately. Maybe add index information in the dropdown somewhere if at all

We can also expand the path item card width. It just means we can't show as many steps concurrently in our target viewport size. If we can include all these things, and maintain our 35 character limit on path item names, I think we could still display at least 4 items in our target viewport size of 1280px wide. Plus that wouldn't be any kind of regression from the previous paths visualization - we'll still be able to reliably show 4 steps in our target viewport. 

Edit - for what it's worth, I do think the step counts provide a value when the path visualization has lots of steps. Imagine being scrolled part way through a longer path viz and not knowing how far you are into the sequence? The scrollbars will indicate this somewhat, but not with any amount of precision. If we can find a way to show all these things, then I think it's worth our time. I can work on figuring out sizes and such in figma if that's useful.

**Comment by EDsCODE** [comment]
hmm @liyiy can you wrap this into #6163 and put the number to the left of the dropdown button? I think we should try to keep the same card width and just squeeze the label slightly more

File: frontend/src/scenes/paths/NewPaths.js [hunk 1]
```diff
@@ -230,11 +230,11 @@ export function NewPaths({ dashboardItemId = null, filters = null, color = 'whit
             })
     }
 
-    const dropOffValue = (pathItemCard) => {
+    const getDropOffValue = (pathItemCard) => {
         return pathItemCard.value - pathItemCard.sourceLinks.reduce((prev, curr) => prev + curr.value, 0)
     }
 
-    const completedValue = (sourceLinks) => {
+    const getCompletedValue = (sourceLinks) => {
```
Comment (neilkakkar) [inline frontend/src/scenes/paths/NewPaths.js:237]: If you want this to be the number of people who reached the node, then it's `targetLinks`, or simply `pathItemCard.value`

**Pull request merged**